### DISCUSSION TITLE
Github Enterprise Cloud in UAE

### DISCUSSION THREAD

**Discussion by galamuri** [community/community#127162] [Enterprise] [answered]
### Body

Does Github Enterprise Cloud have server locations in UAE (Data residency requirements) as we want to upgrade to Cloud. We are currently trying to decide between Enterprise On Prem and Cloud

**Comment by queenofcorgis** [comment, marked as answer]
Hi @galamuri, 

The best place to go for answers to a question like this one is our [Support page](https://support.github.com/contact). Please open a ticket with them for assistance here. Thank you!
